src/content/en/2021/webassembly.md

```diff
@@ -2,7 +2,7 @@
 #See https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Authors'-Guide#metadata-to-add-at-the-top-of-your-chapters
 title: WebAssembly
 description: TODO
-authors: []
+authors: [RReverser]
 reviewers: []
 analysts: []
```

Comment:
@RReverser can you suggest edits to the Authors, Reviewers, and Analysts properties so that they contain everyone who contributed to the chapter, roughly ordered by those with the most significant contributions first? This is what readers will see at the top of the chapter so we want to be sure to give appropriate credit to everyone who helped.

Reply:
I want to add people to reviewers, but at those twitter handles or internal almanac usernames or?

Reply:
GitHub usernames as that's what we mostly use. Make one up for those not on GitHub 😁

Reply:
Ah thanks, makes sense.

```suggestion
authors: [RReverser]
reviewers: [jsoverson, carlopi, azakai]
analysts: []
```